2 - RTB/Documentazione esterna/Piano di Progetto/Piano di Progetto.typ

```diff
@@ -702,5 +702,70 @@ A causa dei rallentamenti incontrati, alcuni ruoli hanno visto un monte ore effe
 
 - Verificatore: i numerosi merge conflicts hanno portato a notevoli rallentamenti in fase di review.
 
+=== Burndown-chart
+[da inserire]
+
+
+== Quinto Sprint dal 04-12-2023 al 11-12-2023
+
+=== Obiettivi raggiunti
+- miglioramento e aggiornamento documento Analisi dei Requisiti;
+- realizzazione mini PoC relativo alla creazione e posizionamento di scaffali dell'ambiente di lavoro;
+- realizzazione mini PoC relativo alla lettura e utilizzo di file SVG e comunicazione con Database;
+- incontro con Proponente in data 06-12-23;
```

Comment:
rimuoverei "mini" e aggiungerei il poc dockerizzato (o dockerizzazione del poc in sostanza)